### DISCUSSION TITLE
error in compile

### DISCUSSION THREAD

**Discussion by sbahamondezp** [NOAA-EMC/WW3#1270] [Q&A]
Hi, 
I have compilation problems
I work on a server that has gfortran 4.8.5 installed (f90 compatible)
The compiler and libraries NetCDF are loaded as follows:

module purge
module load gcc/8.3.0
module load netcdf/4

I set environment variables NetCDF

export WWATCH3_NETCDF=NC4
export NETCDF_CONFIG=/usr/local/bin/nc-config

I use switch Ifremer2, comp y link Gnu 

./w3_clean -c
./w3_setup ../../model -c Gnu -s Ifremer2

 Environment variables for wavewatch III

WWATCH3_LPR      printer
WWATCH3_F90      gfortran
WWATCH3_CC       gcc
WWATCH3_DIR      /home/WW3-6.07.1/model
WWATCH3_TMP     /home/WW3-6.07.1/model/tmp
WWATCH3_SOURCE   yes
WWATCH3_LIST     yes

Finally, when I use w3_automake, the following appears 

Processing ww3_prnc

ad3 : processing w3nmlprncmd
ad3 : processing ww3_prnc
mv: cannot stat ‘ww3_prnc.o’: No such file or directory
      Linking ww3_prnc
      *** file ww3_prnc.o not found ***
*** Missing object files ***
make: *** [/home/WW3-6.07.1/model/exe/ww3_prnc] Error 3

**Comment by mickaelaccensi** [comment]
please download the last version of the develop branch and compile it using cmake. The quick start guide is in WW3/model/README.md

Best,
